pkg/resources/management.cattle.io/v3/clusterroletemplatebinding/ClusterRoleTemplateBinding.md

```diff
@@ -0,0 +1,28 @@
+## Validation Checks
+
+### Escalation Prevention
+
+Users can only create/update ClusterRoleTemplateBindings which grant permissions to RoleTemplates which rights less than or equal to those they currently possess. This is to prevents privilege escalation. 
```

Comment:
```suggestion
Users can only create/update ClusterRoleTemplateBindings which grant permissions to RoleTemplates with rights less than or equal to those they currently possess. This is to prevent privilege escalation. 
```

Reply:
Change Made.